BlazarUI/app/scripts/components/shared/SearchFilter.jsx

```diff
@@ -40,14 +42,23 @@ class SearchFilter extends Component {
     this.refs.searchFilterInput.focus();
   }
 
-  handleChange() {    
+  handleChange() {
     this.setState({
       searchValue: this.refs.searchFilterInput.value
     });
     this.handleSearchDebounced();
   }
 
-  render() {    
+  handleKeyup(e) {
+    const notFocusedOnInputElement = e.target === document.body || e.target.tagName === 'A';
```

Comment:
it's a little tricky figuring out when this shortcut should be enabled. the tagName part is needed to make sure that the shortcut still works after clicking on a sidebar link.